Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/WebRequestBuilder.kt

```diff
@@ -214,6 +216,8 @@ class WebRequestBuilder(
         val submissionPayload = KeyExportFormat.SubmissionPayload.newBuilder()
             .addAllKeys(keyList)
             .setPadding(ByteString.copyFromUtf8(fakeKeyPadding))
+            .addAllVisitedCountries(visitedCountries)
+            .setConsentToFederation(consentToFederation)
```

Comment:
note: the added fields have an impact on plausible deniability. It needs to be checked how this is compensated in the fake submission request

Reply:
true, will check back with the PO's on how to proceed here